pkg/acrn/Dockerfile

```diff
@@ -1,4 +1,5 @@
 # syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+# Dockerfile to build acrn
```

Comment:
I think it is worth adding license/ copyright header for all files while we are here
```
# Copyright (c) 2023 Zededa, Inc.
# SPDX-License-Identifier: Apache-2.0
```

Reply:
Sure, that is even cleaner.